spec/support/matchers/have_field.rb

```diff
@@ -0,0 +1,39 @@
+##
+# Matcher to allow the following forms:
+#
+# expect(an_entry_data_class).to have_field('My Field Name')
+# expect(an_entry_data_class).to have_field('My Field Name').validated_by(:presence)
+# expect(an_entry_data_class).to have_field('My Field Name').with_activemodel_type(:string)
+# expect(an_entry_data_class).to have_field('My Field Name')
+#   .validated_by(:presence, case_insensitive_inclusion: { in: %w[Y N]} )
+# expect(an_entry_data_class).to have_field('My Field Name')
+#   .validated_by(:presence)
+#   .not_validated_by(:ingested_numericality)
+RSpec::Matchers.define :have_field do |field_name|
```

Comment:
I don't know enough about custom Rspec matchers to make sense of this, but nice work on creating this, it makes the language specs super easy to follow 💯